### PR TITLE
Adding PropertyGridViewAccessibleObject unit tests

### LINKED ISSUE #1832: Implement unit tests to cover PropertyGridView control accessibility

### PR DESCRIPTION
**Pull request by vladimir-krestov**
Fixes #1832

## Proposed changes

- Add unit tests for PropertyGridViewAccessibleObject
- Move PropertyGrid tests to PropertyGridAccessibleObjectTests class

## Customer Impact

- No

## Regression? 

- No

## Risk

- Minimal

## Test environment(s) 

- SDK Version: 5.0.100-alpha1-016143
- Microsoft Windows [Version 10.0.18363.592]

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/dotnet/winforms/pull/2875)

### CODE REVIEW

File: src/System.Windows.Forms/tests/UnitTests/AccessibleObjects/PropertyGridViewAccessibleObjectTests.cs [hunk 1]
```diff
@@ -2,46 +2,266 @@
...
+            Assert.True((bool)accessibleObject.GetPropertyValue((UiaCore.UIA)propertyId));
         }
 
         [Theory]
-        [InlineData((int)UiaCore.UIA.GridItemPatternId)]
-        [InlineData((int)UiaCore.UIA.TableItemPatternId)]
-        public void PropertyGridViewAccessibleObject_Supports_TablePattern(int pattern)
+        [InlineData((int)UiaCore.UIA.TablePatternId)]
+        [InlineData((int)UiaCore.UIA.GridPatternId)]
+        public void PropertyGridViewAccessibleObject_IsPatternSupported(int patternId)
```
Comment (RussKie) [inline src/System.Windows.Forms/tests/UnitTests/AccessibleObjects/PropertyGridViewAccessibleObjectTests.cs]: Curious, can this be:
```cs
        [InlineData(UiaCore.UIA.TablePatternId)]
        [InlineData(UiaCore.UIA.GridPatternId)]
        public void PropertyGridViewAccessibleObject_IsPatternSupported(UiaCore.UIA patternId)
```

**Comment by RussKie** [comment]
Tests fail:
```
    System.Windows.Forms.Tests.AccessibleObjects.PropertyGridViewAccessibleObjectTests.PropertyGridViewAccessibleObject_Bounds_InsidePropertyGridBounds [FAIL]
      Assert.True() Failure
      Expected: True
      Actual:   False
      Stack Trace:
        D:\a\1\s\src\System.Windows.Forms\tests\UnitTests\AccessibleObjects\PropertyGridViewAccessibleObjectTests.cs(122,0): at System.Windows.Forms.Tests.AccessibleObjects.PropertyGridViewAccessibleObjectTests.PropertyGridViewAccessibleObject_Bounds_InsidePropertyGridBounds()
```

File: src/System.Windows.Forms/tests/UnitTests/AccessibleObjects/PropertyGridViewAccessibleObjectTests.cs (hunk 1 above)
Comment (vladimir-krestov) [reply src/System.Windows.Forms/tests/UnitTests/AccessibleObjects/PropertyGridViewAccessibleObjectTests.cs]: `UiaCore` class is `internal` but the method is `public`
image

**Comment by vladimir-krestov** [comment]
Fixed the test

**Pull request merged**